CP-CPS.md

```diff
@@ -473,7 +473,11 @@ ISRG may administratively revoke certificates if it determines that the Subscrib
 
 ### 4.9.3 Procedure for revocation request
 
-Anyone may make revocation requests, at any time, via the Certificate Revocation interface of the ACME Protocol defined in RFC 8555 section 7.6. Successful revocation requests with a reason code of `keyCompromise` will result in the affected key being blocked for future issuance and all currently valid certificates with that key will be revoked, regardless of whether compromise was demonstrated per the requirements in Section 4.9.12 of this document.
+Anyone may make revocation requests, at any time, via the Certificate Revocation interface of the ACME Protocol defined in RFC 8555 section 7.6.
+
+A successful revocation request signed by the private key corresponding to the public key in the certificate, per the requirements in Section 4.9.12 of this document, will result in the certificate being revoked with reason code `keyCompromise` and the affected key being blocked for future issuance. All currently valid certificates with that key will also be revoked.
+
+A successful revocation request where a subscriber has demonstrated control over all domains in a certificate and has requested revocation with reason code `keyCompromise` may result in the certificate being revoked with reason code `keyCompromise` but will not result in blocking the key for future issuance or revoking other certificates with the same key unless proof of possession of the private key is demonstrated per the requirements in Section 4.9.12 of this document.
```

Comment:
The paragraph above this feels really nice and concise. I've reworked this one to feel about the same. Feel free to disregard.

```suggestion
If a subscriber has demonstrated control over all domains in a certificate and requests revocation citing `keyCompromise`, the certificate may, at our discretion, be revoked with this reason code. However, this action won't block the key for future issuance nor revoke other certificates using the same key, unless proof of possession of the private key is provided as outlined in Section 4.9.12.
```

Reply:
It does seem important to maintain `a successful revocation request`.